test/config-next/wfe.json

```diff
@@ -14,13 +14,16 @@
     "acceptRevocationReason": true,
     "allowAuthzDeactivation": true,
     "debugAddr": "localhost:8000",
+    "raService": {
+      "serverAddresses": ["boulder:9094"],
+      "serverIssuerPath": "test/grpc-creds/minica.pem",
+      "clientCertificatePath": "test/grpc-creds/boulder-client/cert.pem",
+      "clientKeyPath": "test/grpc-creds/boulder-client/key.pem",
+      "timeout": "90s"
```

Comment:
This is copied from the gRPC client configs ther RA already had. Looking at other configs this seems to be the standard we chose for gRPC configs in general, if you think we should switch all of the others over as well I'd be happy to open a new issue and writing a specific PR for that.